vision/text/detect_words.py

```diff
@@ -99,7 +105,12 @@ def _get_rotated_min_area_rect(
             blur_image[:, :, 2],
         )  # separate channels
 
-        isblue = np.logical_and((b_image - r_image > 120), (b_image - g_image > 20))
+        r_image = np.where(r_image == 0, 1, r_image)  # prevent division by zero
+        g_image = np.where(r_image == 0, 1, r_image)  # prevent division by zero
+
+        isblue = np.logical_and(
+            (b_image / r_image > RED_WT), (b_image / g_image > GREEN_WT)
```

Comment:
It looks like this would count near-black colors in the image as blue. For example if the RGB pixel value is (1, 1, 2), it would be counted as blue even though it is basically black. Should add a minimum blue threshold of 20 or so to this `and` statement, so that there would have to be a minimum blue value to be counted as blue.